__test__/fixtures/rawSierraAccountData.js

```diff
@@ -530,10 +530,7 @@ export const patron = {
   barcodes: ["23333121538324"],
   expirationDate: "2025-03-28",
   emails: ["streganonna@gmail.com", "spaghettigrandma@gmail.com"],
-  homeLibrary: {
-    code: "sn",
-    name: "Stavros Niarchos Foundation Library (SNFL)",
-  },
+  homeLibraryCode: "sn   ",
```

Comment:
Are the extra spaces expected?

Reply:
yea. i think it's a fixed field, so location codes are 5 characters.